requirements/lit/docs.txt

```diff
@@ -0,0 +1,7 @@
+-r ../docs.txt
+
+ipython[notebook]
+ipython_genutils
+
+sphinx-autobuild
+https://storage.googleapis.com/grid-packages/lightning-ai-sphinx-theme/build-31.3.zip
```

Comment:
This will tell it to use the "Pytorch Lightning" theme (without the lightning.ai header) and should unblock this PR. We should update this to use the theme with the lightning.ai header before merging.

```suggestion
https://github.com/PyTorchLightning/lightning_sphinx_theme/archive/master.zip#egg=pt-lightning-sphinx-theme
```